tests/lib/rules/no-await-in-loop.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * @fileoverview Tests for no-await-in-loop.
+ * @author Nat Mote (nmote)
+ */
+
+"use strict";
+
+const rule = require("../../../lib/rules/no-await-in-loop"),
+    RuleTester = require("../../../lib/testers/rule-tester");
+
+/**
+ * A convenience function so we can avoid duplicating the parserOptions everywhere
+ *
+ * @param {string} code: The program text to test
+ * @param {?string} error: The expected error, if any
+ * @returns {Object}: The object that describes the test case to the RuleTester
+ */
+function wrapTest(code, error) {
```

Comment:
This function is fine with me, but one alternative would be to put the defaults in the `RuleTester` constructor:

```js
const ruleTester = new RuleTester({ parserOptions: { ecmaVersion: 2017 } });
```